Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/OnboardingGenerateKeyViewController.swift

```diff
@@ -57,11 +57,21 @@ class OnboardingGenerateKeyViewController: AddKeyOnboardingViewController {
             }
 
             let message = "The key successfully created. Add it to a Safe on desktop and then restart the mobile app."
-
+            
             let passcodeCompletion = { [unowned self, unowned vc] in
                 App.shared.snackbar.show(message: message)
-                let detailsVC = OwnerKeyDetailsViewController(keyInfo: keyInfo, completion: self.completion)
-                vc.show(detailsVC, sender: vc)
+                let backupVC = BackupIntroViewController()
+                backupVC.backupCompletion = { [unowned self, unowned vc] backup in
+                    if backup {
+                        //TODO: proceed to backup screen instead
+                        let detailsVC = OwnerKeyDetailsViewController(keyInfo: keyInfo, completion: self.completion)
```

Comment:
Why not proceed to the backup screen (ExportViewController) right now?

Reply:
it can be done